src/views/Nft/market/Collection/Header.tsx

```diff
@@ -25,8 +23,11 @@ const Header: React.FC<HeaderProps> = ({ collection }) => {
   const { totalSupply, numberTokensListed, totalVolumeBNB, banner, avatar } = collection
   const { t } = useTranslation()
   const { pathname } = useLocation()
-  const owners = ethers.BigNumber.from(numberTokensListed)
-  const volume = ethers.BigNumber.from(totalVolumeBNB)
+
+  const volume = parseFloat(totalVolumeBNB).toLocaleString(undefined, {
```

Comment:
We should have an helper for this tbh (if not already here)

Reply:
yeah its the same in bunch of places, agree, good item to refactor 👍